parsec/mca/device/cuda/transfer.c

```diff
@@ -165,7 +165,7 @@ static const parsec_symbol_t symb_CUDA_d2h_task_param = {
     .flags = 0x0
 };
 
-int32_t parsec_CUDA_d2h_max_flows = 0;
+extern int32_t parsec_CUDA_d2h_max_flows = 0;
```

Comment:
We shouldn't mark variables as `extern` in compilation units. Can we put them in a header?

Reply:
This is fixed in master (exactly as you propose), I'm just taking the path of least resistance here to hot fix v3.